docs/specifications/tests/Consistency-TP/consistency06.md

```diff
@@ -0,0 +1,80 @@
+# CONSISTENCY06: SOA MNAME consistency
+
+## Test case identifier
+
+**CONSISTENCY06**
+
+## Objective
+
+All authoritative name servers must serve the same SOA record (section
+4.2.1) of [RFC 1034] for the tested domain. As per section 3.3.13 of 
+[RFC 1035] the MNAME field in the SOA RDATA refers to the name of 
+"the name server that was the original or primary source of data 
+for this zone". Inconsistency in MNAME of the domain might result in 
+operational failures for applications that uses MNAME.
+
+## Inputs
+
+* The domain name to be tested ("child zone")
+
+## Ordered description of steps to be taken to execute the test case
+
+ 1. Obtain the list of name server IPs for the *child zone* from [Method4] 
+    and [Method5].
+ 2. Create an SOA query for *child zone* apex and send it to all name 
+    server IPs.
+ 3. Retrieve the SOA RR from the responses from all name server IPs.
+ 4. If a name server does not respond, emit *[NO_RESPONSE]*.
+ 5. If a name server responds but does not include a SOA record in 
+    the response, emit *[NO_RESPONSE_SOA_QUERY]*.
```

Comment:
How could you retrieve the SOA RR if was none? Maybe put this step before the SOA RR retrieval step?

Reply:
Updated in new commit.